server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/InfinispanLogger.java

```diff
@@ -28,9 +28,7 @@
 import org.jboss.logging.annotations.Message;
 import org.jboss.logging.annotations.MessageLogger;
 
-import static org.jboss.logging.Logger.Level.DEBUG;
-import static org.jboss.logging.Logger.Level.INFO;
-import static org.jboss.logging.Logger.Level.WARN;
+import static org.jboss.logging.Logger.Level.*;
```

Comment:
Yes, that has been fixed in the latest formatter: https://github.com/infinispan/infinispan/pull/4439